dash-renderer/src/observers/storedCallbacks.ts

```diff
@@ -0,0 +1,69 @@
+import {
+    concat,
+    filter,
+    groupBy,
+    isNil,
+    partition,
+    reduce,
+    toPairs
+} from 'ramda';
+
+import { IStoreState } from '../store';
+
+import {
+    aggregateCallbacks,
+    removeStoredCallbacks
+} from '../actions/callbacks';
+
+import {
+    ICallback,
+    IStoredCallback
+} from '../types/callbacks';
+
+import { getPendingCallbacks } from '../utils/callbacks';
+import { IStoreObserverDefinition } from '../StoreObserver';
+
+const observer: IStoreObserverDefinition<IStoreState> = {
+    observer: ({
+        dispatch,
+        getState
+    }) => {
+        const { callbacks } = getState();
+        const pendingCallbacks = getPendingCallbacks(callbacks);
```

Comment:
Stored callbacks are kept to ensure callbacks with dependencies have access to the required information (eg. layout update). Callbacks without an execution group are dropped immediately while callbacks with an execution group are dropped out when no other callbacks remain pending for the group.